### PR TITLE
 Changed 'vm docker create' command to use new Docker extension

### PR DESCRIPTION
**Pull request by ruslangabitov**
cc @ahmetalpbalkan

### CODE REVIEW

**Comment by anuchandy** [comment]
LGTM

File: lib/commands/arm/vm/vm._js [hunk 1]
```diff
@@ -456,7 +456,7 @@ exports.init = function (cli) {
       .usage('[options] <resource-group> <name> <location> <os-type>')
       .option('-g, --resource-group <resource-group>', $('the name of the resource group'))
       .option('-n, --name <name>', $('the virtual machine name'))
-      .option('-T, --docker-port <port>', $('Port to use for docker [4243]'))
+      .option('-T, --docker-port <port>', $('Port to use for docker [2376]'))
```
Comment (ahmetalpbalkan) [inline lib/commands/arm/vm/vm._js]: Hmm should we keep this at 4243, we'll probably have to update our docs and if there are scripts relying on this they're gonna break.

Comment (ruslangabitov) [inline lib/commands/arm/vm/vm._js]: Ok, I'll revert that.

**Comment by ahmetalpbalkan** [comment]
hey @ruslangabitov sorry for that comment, turns out 4243 was a historical decision in Docker that collided with a IANA registered port number. https://github.com/docker/docker/issues/1440

Docker docs suggest 2376 for SSL connections now and that's what's registered in IANA. http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=docker 

I guess we historically had 4243 when we started as well, let's change it.

**Comment by ruslangabitov** [comment]
@ahmetalpbalkan, I've removed my last commit, now default port is 2376.

**Comment by ahmetalpbalkan** [comment]
Lgtm thanks.

**Comment by ahmetalpbalkan** [comment]
Any guesses when this would make it to the released version and as which version? I will probably need to update the docs too.

**Comment by ruslangabitov** [comment]
@ahmetalpbalkan looks like it will be a part of sprint50, and we will send PR to Azure's repo at the end of June (23rd).

**Comment by ahmetalpbalkan** [comment]
Hmm that looks a bit late than I expected. What CLI version is this going to be? Given the fact that the change is considerably small, I'd like to get it to the next version of the CLI that still accepts PRs.

**Comment by anuchandy** [comment]
cc : @kmouss @davidni @babumuralidharan 

@kmouss what i understood in during today's meeting is next CLI release is close to 23rd June. Just confirming is there any other release before that where we can push this fix?

**Pull request merged**